techniques/aria/ARIA1.html

```diff
@@ -186,25 +181,25 @@ <h2>Related Resources</h2>
 
 
 <section id="related">
-      <h2>Related Techniques</h2>
+      <h2>関連テクニック</h2>
       <ul>
-         <li><a href="../aria/ARIA2">ARIA2: Identifying a required field with the aria-required property</a></li>
-         <li><a href="../aria/ARIA7">ARIA7: Using aria-labelledby for link purpose</a></li>
+         <li><a href="../aria/ARIA2">ARIA2: aria-required プロパティによって必須項目を特定する</a></li>
```

Comment:
先行コメントの理由で原文に書き戻します。
```suggestion
         <li><a href="../aria/ARIA2">ARIA2: Identifying a required field with the aria-required property</a></li>
```